input/docs/cs/affine_picture_transformation.md

```diff
@@ -0,0 +1,34 @@
+﻿title: Afinní transformace obrázku pomoci rotace
```

Comment:
Spíš: Rotace je jednou z afinních transformací, tím pádem to nedává moc smysl. co třeba "Rotace obrázku pomocí transformační matice"?